package.json

```diff
@@ -11,7 +11,8 @@
     "static": "node ./serve/static",
     "lint": "eslint",
     "test": "jest",
-    "generate": "cd ../../generator && yarn install && yarn generate"
+    "generate": "cd ../../generator && yarn install && yarn generate",
+    "postinstall": "cd serve && yarn install"
```

Comment:
Would preferable run this with `--frozen-lockfile`, but was running into some issues with that. Need to look into it more.